client/src/app/__tests__/AppParentSpec.js

```diff
@@ -156,6 +156,46 @@ describe('<AppParent>', function() {
     });
 
 
+    it('should set log closed by default', function(done) {
+
+      // given
+      const backend = new Backend({
+        sendReady() {
+          expect(appParent.getApp().state.layout).to.eql({
+            panel: {
+              open: false,
+              tab: 'log'
+            }
+          });
+
+          done();
```

Comment:
Let's use this pattern so that we don't need to wait for timeout in case assertion breaks:

```suggestion
          try {
            expect(appParent.getApp().state.layout).to.eql({
              panel: {
                open: false,
                tab: 'log'
              }
            });
  
            done();
          } catch (error) {
            done(error);
          }
```